README.md

```diff
@@ -9,7 +9,7 @@ Integrating Site Recording to your website doesn’t require any coding with Goo
 
 1. Log in to Google Tag Manager, navigate your account and click on "Add a new tag".
 
-2. Search for our community template "Site Recording by 500 apps " in the "Community Template Gallery".
+2. Search for our community template "Site Recording by 500 apps" in the "Community Template Gallery".
```

Comment:
Is it Site Recording or Siterecording or SiteRecording

check with website and marketing team too